db/migrate/20241209191600_add_user_tags.rb

```diff
@@ -0,0 +1,11 @@
+class AddUserTags < ActiveRecord::Migration[7.2]
+  def change
+    create_table :user_tags do |t|
+      t.integer :user_id, null: false
+      t.integer :tag_id, null: false
+      t.timestamps null: true
```

Comment:
Huh, do timestamps need to be nullable? (It seems like we should specify this, which makes sense, but https://guides.rubyonrails.org/active_record_migrations.html#migration-overview suggests that `null: false` should work for timestamps too, and seems preferable)

Reply:
I think most of our other timestamps _are_ nullable, but they also have `precision: nil` defined on them - my guess is some defaults may have changed in different Rails versions.

Did you use a specific rationale to pick this config? Wondering if I'm missing something

Reply:
No clue, I just c/p the other _tags

Reply:
Ah ok, I think it's ok then!